### PR TITLE
feat(ui): create welcome page

### PR DESCRIPTION
**Pull request by ronald-luo**
image

image

### CODE REVIEW

**Comment by KaiKitJeffreyChan** [comment]
Everything looks good, few things to add. The ground is not a static image and is actually an animation using a few tailwind classes. Please also implement the mobile version which is also on the Figma. Good work so far!

**Comment by KaiKitJeffreyChan** [comment]
- Button size is too big, no hover on buttons
- Icons should be stuck to bottom
- Text color on nav wrong
- text color in paragraph is wrong
- line spacing for text is wrong
- nav bar spacing is wrong

File: src/components/SocialButtons.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import Image from "next/image";
+
+export default function SocialButtons() {
```
Comment (KaiKitJeffreyChan) [inline src/components/SocialButtons.tsx:3]: Change to const SocialButtons = () => { and then export default at the bottom just to keep a convention

**Review by Krish120003 — APPROVED** [review]

**Review by KaiKitJeffreyChan — APPROVED** [review]
noice lgtm

**Pull request merged**